.claude/hooks/tests/test_git_header_statusline.py

```
@@ -0,0 +1,254 @@
+    def test_red_no_remote_with_uncommitted(self, temp_git_repo, git_header_script):
+        """RED: Test no remote with uncommitted changes"""
+        # Create uncommitted changes
+        with open("test_file.txt", "w") as f:
+            f.write("uncommitted content")
+
+        stdout, stderr, returncode = self.run_git_header(git_header_script)
+
+        # Should show both no remote and uncommitted
+        assert "(no remote +uncommitted)" in stdout
+
+    def test_red_ahead_commits_status(self, temp_git_repo, git_header_script):
```

Ruff (0.13.1)

Unused method argument: `temp_git_repo`

(ARG002)